src/openrct2/park/ParkFile.cpp

```diff
@@ -2312,6 +2313,31 @@ int32_t scenario_save(u8string_view path, int32_t flags)
     catch (const std::exception& e)
     {
         log_error(e.what());
+
+        Formatter ft;
+        ft.Add<const char*>(e.what());
+        context_show_error(STR_FILE_DIALOG_TITLE_SAVE_SCENARIO, STR_STRING, ft);
+        gfx_invalidate_screen();
+
+        auto ctx = OpenRCT2::GetContext();
+        auto uictx = ctx->GetUiContext();
+
+        std::string title = "Error while saving";
+        std::string message
+            = "There was an error while saving scenario.\nhttps://github.com/OpenRCT2/OpenRCT2/issues/17664\nWe would like to "
+              "collect more information about this issue, if this did not happen due to missing permissions, lack of space, "
+              "etc. please consider submitting a bug report. To collect information we would like to trigger an assert.";
+
+        std::string report_bug_button = "Report bug, trigger an assert";
+        std::string skip_button = "Skip reporting, let me continue";
+
+        std::vector<std::string> buttons{ report_bug_button, skip_button };
```

Comment:
I think this will cause `std::string` copy, rather than a move.

Reply:
How would I move it?

Reply:
`std::move`